pkg/interop/errors.go

```diff
@@ -48,26 +48,32 @@ var (
 	ErrFrameSize          = defineError("frame_size", ResultFrameSizeError, "frame size error")
 )
 
-var errorResults = make(map[string]Result)
+var errorResults = make(map[string]ResultCode)
 
-func defineError(name string, result Result, message string) errors.Definition {
+func defineError(name string, result ResultCode, message string) errors.Definition {
 	definition := errors.DefineInvalidArgument(name, message)
 	errorResults[definition.FullName()] = result
 	return definition
 }
 
+type errorMessage struct {
+	Message string `json:"message"`
+}
+
 // ErrorHandler is an echo.HTTPErrorHandler.
 func ErrorHandler(err error, c echo.Context) {
 	if httpErr, ok := err.(*echo.HTTPError); ok {
 		c.JSON(httpErr.Code, httpErr.Message)
 		return
 	}
 
-	result := ResultOther
-	statusCode := http.StatusInternalServerError
+	result, statusCode, description := ResultOther, http.StatusInternalServerError, ""
```

Comment:
:frowning_face: 
This refactoring does not seem to help readability, please consider just declaring an additional `var description string` below the previous lines

Reply:
On one hand I agree that this isn't perfect. `""` signals that it's explicitly `""`, while declaring a `var description string` signals that we'll assign it later. However, I don't think this is important enough to fix. We all understand what's going on.